modes/unimpaired/evil-collection-unimpaired.el

```diff
@@ -112,17 +112,57 @@
   (ignore count)
   (evil-collection-unimpaired--encode beg end #'url-unhex-string))
 
+;; https://stackoverflow.com/questions/2423834/move-line-region-up-and-down-in-emacs
+(defun evil-collection-unimpaired--move-text (arg)
+  "Move text down if ARG is positive, otherwise move text up."
+  (cond
+   ((and mark-active transient-mark-mode)
+    (if (> (point) (mark))
+        (exchange-point-and-mark))
+    (let ((column (current-column))
+          (text (delete-and-extract-region (point) (mark))))
+      (forward-line arg)
+      (move-to-column column t)
```

Comment:
Style: I prefer to replace `t` with `:force` to make it more readable, without looking at the function signature.